newrelic_lambda_cli/integrations.py

```diff
@@ -517,12 +568,12 @@ def install_log_ingestion(
     Returns True for success and False for failure.
     """
     assert isinstance(input, IntegrationInstall)
```

Comment:
Maybe add something here warning users that the old log ingestion function is present in their region and they should delete it:
```suggestion
    assert isinstance(input, IntegrationInstall)
    function = get_function(input.session, "newrelic-log-ingestion")
    if function:
        warning("It looks like an old log ingestion function is present in this region. Consider manually deleting this as it is no longer used and has been replaced by a log ingestion function specific to the stack.")
```

Reply:
addressed in the revised PR